getconf/base.py

```diff
@@ -322,14 +234,36 @@ def list_keys(self):
     def get_ini_template(self):
         """Export commented INI file content mapping to the defaults"""
         section_to_keys = collections.defaultdict(list)
-        for key in self.seen_keys:
-            section_to_keys[key.section].append(key)
+        for config_key in self.seen_keys:
+            section, entry = self._parser_finder.split_section_key(config_key.key)
+            envvar = self._env_finder.env_key(config_key.key)
+
+            # Remove any new line in doc
+            one_line_doc = ' '.join(config_key.doc.split())
+            doc_part = (' - %s' % one_line_doc) if one_line_doc else ''
+
+            if config_key.type_hint == 'list':
+                default = ', '.join(config_key.default)
+            elif config_key.type_hint == 'bool':
+                default = 'on' if config_key.default else 'off'
+            else:
+                default = str(config_key.default)
+            default_str = ' %s' % default if default else ''
+
+            entry_str = '; {envvar} - type={type_hint}{doc_part}\n;{entry} ={default_str}'.format(
+                envvar=envvar,
+                type_hint=config_key.type_hint,
+                doc_part=doc_part,
+                entry=entry,
+                default_str=default_str,
+            )
+            section_to_keys[section].append((entry, entry_str))
 
         parts = []
         for section, keys in sorted(section_to_keys.items()):
             parts.append('[%s]' % section)
-            for config_key in sorted(keys, key=lambda k: k.entry):
-                parts.append(config_key.as_ini_entry())
+            for _entry, entry_str in sorted(keys, key=lambda entry_entry_str: entry_entry_str[0]):
```

Comment:
`operator.itemgetter` 😢 

What is supposed to be `entry_entry_str` that is not `entry_str` ?